.github/ci/.pylintrc

```diff
@@ -41,7 +41,7 @@ enable=indexing-exception,old-raise-syntax
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 # TODO(b/154672575): Enable wrong-import-order.
-disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,no-member,no-name-in-module,import-error,unsubscriptable-object,unbalanced-tuple-unpacking,undefined-variable,not-context-manager,useless-object-inheritance,unnecessary-pass,no-else-return,no-else-raise,no-else-continue,wrong-import-order,no-value-for-parameter,misplaced-comparison-constant,no-init,abstract-method,unpacking-non-sequence
+disable=design,similarities,no-self-use,attribute-defined-outside-init,locally-disabled,star-args,pointless-except,bad-option-value,global-statement,fixme,suppressed-message,useless-suppression,locally-enabled,no-member,no-name-in-module,import-error,unsubscriptable-object,unbalanced-tuple-unpacking,undefined-variable,not-context-manager,useless-object-inheritance,unnecessary-pass,no-else-return,no-else-raise,no-else-continue,wrong-import-order,no-value-for-parameter,misplaced-comparison-constant,no-init,abstract-method,unpacking-non-sequence,super-with-arguments
```

Comment:
If you're adding this just because of the sklearn example, I can fix the super issues and remove this in a separate PR.

Reply:
I did, I also added it as it's a bit to restrictive imo. I use both super styles most of the times. That said, if we want to keep in sync w TFX repo, we should remove it